File: submissions/example-santa/tree.rb
def print_tree(height)
  height.times do |i|
    puts (' ' * (height - i - 1)) + ('*' * (2 * i + 1))
  end
  puts (' ' * (height - 1)) + '|'
end

print_tree(5)
puts "\nMerry Christmas! 🎄"

File: submissions/example-santa/tree.c
#include <stdio.h>

void print_tree(int height) {
    for (int i = 0; i < height; i++) {
        for (int j = 0; j < height - i - 1; j++) printf(" ");
        for (int k = 0; k < 2 * i + 1; k++) printf("*");
        printf("\n");
    }
    for (int i = 0; i < height - 1; i++) printf(" ");
    printf("|\n");
}

int main() {
    print_tree(5);
    printf("\nMerry Christmas! 🎄\n");
    return 0;
}

